common/base/src/logging.cpp

```diff
@@ -27,66 +29,67 @@ namespace genomeworks
 
 namespace logging
 {
-static std::shared_ptr<spdlog::logger> logger = nullptr;
+static std::unique_ptr<std::ostream> out_stream_ = nullptr;
+static std::unique_ptr<std::ofstream> out_file_  = nullptr;
```

Comment:
`out_file_` is not needed anymore.

Reply:
my bad, removed it now

Reply:
Did I submit this review already?! Well, ok.. It's fixed now....